src/utils/slangHelper.ts

```diff
@@ -125,9 +125,9 @@ export const externalBuiltIns = {
  */
 export function createContext<T>(
   chapter: number,
-  variant: Variant | undefined,
   externals: string[],
-  externalContext: T
+  externalContext: T,
+  variant?: Variant
```

Comment:
Instead of having variant be union of _Variant_ and _undefined_, have it be an optional property